pkg/slo/query/query_test.go

```diff
@@ -17,7 +17,7 @@ var _ = Describe("Converting ServiceLevelObjective Messages to Prometheus Rules"
 
 	When("We define pre-configured metrics in def.go", func() {
 		It("Should produce valid templates and regex at compile time", func() {
-			Expect(query.AvailableQueries).To(HaveLen(3))
+			Expect(query.AvailableQueries).ToNot(HaveLen(0)) // > 0 available pre configured metrics
```

Comment:
nit: use `NotTo` instead of `ToNot` (autocomplete always suggests ToNot for some reason)

Reply:
Done